### PR TITLE
jt member photo

### LINKED ISSUE #160: Add member image for Jacqueline Thai
This is an issue.

### PR DESCRIPTION
**Pull request by ctc-devops**
This is an issue.
closes #160

### CODE REVIEW

**Review by Vakhshoori101 — COMMENTED** [review]
LGTM, last thing is to fix the merge conflict. DM me on slack if you need help

File: website/src/app/views/about/members.js [hunk 1]
```diff
@@ -122,6 +123,7 @@ import ruslan from '../../../images/about/memberImages/ruslan.jpg';
 import derek from '../../../images/about/memberImages/derek.jpg';
 import duongvu from '../../../images/about/memberImages/duongvu.jpg';
 
+
```
Comment (Vakhshoori101) [inline website/src/app/views/about/members.js:126]: remove this empty line

File: website/src/app/views/about/members.js [hunk 2]
```diff
@@ -395,11 +397,18 @@ const members = {
...
+      position: 'Developer',
+      linkedinURL: 'https://www.linkedin.com/in/jthai-1201/',
+      imageURL: jackie,
+    },
     {
       name: 'Jared Lai',
       position: 'Developer',
       linkedinURL: 'https://www.linkedin.com/in/jared-l-02aa8b126',
       imageURL: jared,
+
```
Comment (Vakhshoori101) [inline website/src/app/views/about/members.js:411]: remove this empty line

**Review by Vakhshoori101 — COMMENTED** [review]
Address the comments then I can merge :)

**Comment by Jacqueline1201** [comment]
ok I think I fixed it! :0

**Review by Vakhshoori101 — APPROVED** [review]
LGTM

**Pull request merged**